speech/infinite_streaming_from_file/README.md

```diff
@@ -0,0 +1,57 @@
+# Google Cloud Speech API Go example
+
+## Authentication
```

Comment:
Do we need a full README for this? If this is going to be a tutorial on cloud.google.com, that should explain all of this. In general, we should avoid duplicating information about how to run the sample. READMEs, unfortunately, fall out of date very easily.

Reply:
This is only a concise readme which includes the key points to run program instead of a full tutorial on website. All the other samples have the same readme so I just keep the same format to make it easier for our customers who is the first time to try to run this program.